### ISSUE TITLE
Scrolling issue in songlist of albums

### ISSUE THREAD

**Issue by Roello09** [timusus/Shuttle#406]
**Shuttle version:**
 
 v2.0.7-beta9
 
**Device, OS:**
 
Samsung A6 2018
Android 8.0.0

**Description of bug:**
 
The song list of an abum gets pulled down when scrolling up, as if the top of an album has been reached. Scrolls up normally if you scroll down first then up in one motion.
 
**Steps to reproduce:**
 
 1. Go to the song list of an album
 2. Scroll upwards without scrolling down first
 3. Bug occurs
 
**Expected outcome:**
 
Should scroll up in the list if the top hasn't been reached yet
 
**Observations/Actual Result:**
 
 The list gets pulled down

**Comment by jugges** [comment]
I can't seem to reproduce this bug in beta 11

**Comment by Roello09** [comment]
Yeah it's fixed for me too.

**Comment by timusus** [comment]
I don't believe I've made any changes to the app to introduce or resolve this change. It sounds like it's either never been solved, and only occurs under certain conditions, or it's not caused by Shuttle..
